pkg/manifestclient/readwrite_roundtripper.go

```diff
@@ -5,16 +5,20 @@ import (
 	"fmt"
 	"io"
 	"io/fs"
-	"k8s.io/client-go/rest"
 	"net/http"
 	"os"
+
+	"k8s.io/client-go/rest"
 )
 
 // RecommendedRESTConfig is meant to be paired with the HTTPClients below
 func RecommendedRESTConfig() *rest.Config {
 	return &rest.Config{
 		QPS:   1000,
 		Burst: 10000,
+		ContentConfig: rest.ContentConfig{
+			ContentType: "application/json",
```

Comment:
Is there a Godoc or similar that people are likely to see when they go to instantiate and use one of the roundtrippers from this package without using the "recommended" config?

Reply:
Yeah, there is, added an extra line to help push folks to using it correctly.

```
// For Kubernetes built in clients, use `manifestclient.RecommendedKubernetesWithClient(httpClient)`
```